src/actions/__tests__/range-test.js

```diff
@@ -3,7 +3,7 @@ import getRange from '../range'
 describe('actions/getRange', () => {
   it('returns range between 2 and totalPage if totalPage' +
```

Comment:
Do this instead of concatenation.
```js
  describe('actions/getRange', () => {
    describe('when totalPage is larger than preset range', () => {
      // BTW, what is preset range?
      it('returns range between 2 and totalPage', () => {
        // ...
      });
    });
  });
```